Dixit_Client/View/LoginWindow.xaml.cs

```diff
@@ -22,6 +22,8 @@ public partial class LoginWindow : Window
         public LoginWindow()
         {
             InitializeComponent();
+            var model = new Dixit_Client.Model.Model();
+            model.Login("Asda");
```

Comment:
I don't think you need it here, loginwindow has its own viewmodel, you can create the model instance there